packages/contracts/deploy/L2/100-Proxies.ts

```diff
@@ -0,0 +1,21 @@
+import { DeployFunction } from 'hardhat-deploy/dist/types'
```

Comment:
Why did you name these files starting with `100-`, `101-`, not `011-`, `012-`?

Reply:
He want to separate with predeployed contracts AFAIK.

Reply:
Yes, I gave them +100 as a separator because they're contracts, not predeploy properties.